packages/template-retail-react-app/package.json

```diff
@@ -20,6 +20,7 @@
     "lint:fix": "npm run lint -- --fix",
     "postinstall": "npm run compile-translations && npm run compile-translations:pseudo",
     "push": "npm run build && pwa-kit-dev push",
+    "push:source-map": "cross-env SSR_SOURCE_MAP=true npm run build && pwa-kit-dev push",
```

Comment:
I'd say `push:ssr-source-map` to be specific

Reply:
Lol, said the opposite over here, but I'm unclear on the "why" behind this

https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1571#discussion_r1409818806